src/pynwb/data/nwb.file.yaml

```diff
@@ -431,28 +410,23 @@ groups:
           and also ImageSegmentation and DfOverF interfaces.'
         quantity: '*'
       quantity: '?'
-  - neurodata_type_inc: DynamicTable
-    name: trials
-    doc: Data about experimental trials
-    datasets:
-    - neurodata_type_inc: TableColumn
-      name: start
-      dtype: float
-      doc: The start time of each trial
-      attributes:
-      - name: description
-        dtype: text
-        doc: Value is 'the start time of each trial'
-        value: the start time of each trial
-    - neurodata_type_inc: TableColumn
-      name: end
-      dtype: float
-      doc: The end time of each trial
-      attributes:
-      - name: description
-        dtype: text
-        doc: Value is 'the end time of each trial'
-        value: the end time of each trial
+  - name: intervals
```

Comment:
@bendichter Can you read over the docs for intervals, epochs, and trials. I made the best attempt I could with my understanding of how they are used, but your input would be good